pkg/gadget-service/api-helpers/apihelpers.go

```diff
@@ -71,11 +71,16 @@ func ToParamDescs(p api.Params) params.ParamDescs {
 	return res
 }
 
+// Validate will validate the given parameter values against the parameter
+// description. If a parameter is not set, the default value will be given.
 func Validate(p api.Params, v api.ParamValues) error {
```

Comment:
It seems a bit off to set the default value on a function called Validate(), perhaps add a new helper that assigns the default value and keep the validate as it was before?